src/view.rs

```diff
@@ -507,18 +507,18 @@ pub fn format_quantity(
 }
 
 fn format_duration(duration: Duration, number_style: impl Display, unit_style: impl Display) -> String {
-    let mut seconds = duration.as_secs();
+    let mut millis = duration.as_millis();
```

Comment:
```suggestion
    let mut milliseconds = duration.as_millis();
```

I prefer fully writing out everything.